### ISSUE TITLE
Voice over is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed : A11y_.NET Core_RecipesAppMyRecipesNew_Search box_screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#22971]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags
#A11yTCS;#A11yMAS;#WCAG1.3.1;#Screenreader;#E2E_.NETCore_Mac_Jun2024;#.NET Core;#VoiceOver;#DesktopApp;#Device;#FTP;#A11ySev2;#Mac;#Element:EditField;#AILimited;#Closed;
## Environment Details:
Application: .NET Core
Version 2 (2)
Operating System: Sonoma 14.5 

## Repro Steps:

1. Turn on Voice over screen reader
2. Install and open ".NET MAUI" app.
3. TAB till "Search recipe" edit filed search some thing 
6. Observe voice over is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed.

## Actual Result:
Voice over is not announcing the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed
## Expected Result:
Voice over should announce the status message as result found when results are found and visible status message when result is not fond for search recipe edit filed.
## User Impact:
Screen reader user will not known for their search result is found or not.
### Attachment

https://github.com/dotnet/maui/assets/124254981/8681a9e6-9e2e-4e32-9d0e-85b4362d0566

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio 17.6.12(build 410) (8.0.40&8.0.21). Can repro on Maccatalyst platform with sample project.

**Comment by kupatkar99** [comment]
GithubTags:#Rev:yaja;

**Comment by Yash14j** [comment]
GithubTags:#Rev:yaja;

**Comment by kapilvaishna** [comment]
Issue is not repro in version 3 so closing this bug.
Attachment:
https://microsoft-my.sharepoint.com/:v:/p/v-kvaishnav/EZ6_Kh94w2dHlg8iHyGjvd0BHZQCLu54_XzTNY-y_bzCrQ?e=MJyTMv
